### PR TITLE
TheMediaGrid Bid Adapter: added support of PBAdSlot module

### PR DESCRIPTION
**Pull request by TheMediaGrid**
## Type of change

- [x] Feature

## Description of change

Added support of PBAdSlot module

- grid-tech@themediagrid.com
- [x] official adapter submission

### CODE REVIEW

File: test/spec/modules/gridBidAdapter_spec.js [hunk 1]
```diff
@@ -428,6 +428,48 @@ describe('TheMediaGrid Adapter', function () {
       expect(payload.tmax).to.equal(3000);
       getConfigStub.restore();
     });
+    it('should contain imp[].ext.data.adserver if availabel', function() {
```
Comment (patmmccann) [inline test/spec/modules/gridBidAdapter_spec.js:431]: available

Comment (TheMediaGrid) [reply test/spec/modules/gridBidAdapter_spec.js]: Fixed

**Comment by TheMediaGrid** [comment]
@patmmccann , @mmoschovas Hello, are there any questions that block the merge?

**Review by mmoschovas — APPROVED** [review]
LGTM

**Pull request merged**